modules/sdk-coin-ada/src/lib/transaction.ts

```diff
@@ -118,7 +118,7 @@ export class Transaction extends BaseTransaction {
    */
   loadInputsAndOutputs(): void {
     const outputs: Entry[] = [];
-    const inputs: Entry[] = [];
```

Comment:
why change to any here?

Reply:
Because Entry type does not support the input format, do we have another besides any?